ZenPacks/zenoss/Docker/parsing.py

```diff
@@ -66,3 +70,27 @@ def rows_from_output(output, expected_columns=None):
             for column, (start, end) in column_indexes.items()})
 
     return rows
+
+
+def cgroup_path_from_output(output):
+    """Return mounted path to cgroup given the content of /proc/self/mountinfo.
+
+    Raises CgroupPathNotFound if the path to cgroup is not found in the file.
+    """
+    lines = output.strip().splitlines()
+    if not lines:
+        return CgroupPathNotFound()
```

Comment:
I think this is supposed to be `raise CgroupPathNotFound()` instead of `return CgroupPathNotFound()`.